.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -159,4 +195,22 @@ kernel_matrix_testing_upload_deps:
       - $LibvirtSSHKeyARM.pub
       - $STACK_DIR
 
-
+kernel_matrix_testing_run_tests:
+  stage: kernel_matrix_testing
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:7c39524b
+  needs: ["kernel_matrix_testing_upload_deps", "kernel_matrix_testing_setup_env"]
+  tags: ["arch:amd64"]
+  variables:
+    AWS_EC2_SSH_KEY_FILE: $CI_PROJECT_DIR/ssh_key
+    GOVERSION: "1.19"
+  before_script:
+    - !reference [.kernel_matrix_testing_new_profile]
+  script:
+    - INSTANCE_IP=$(cat $CI_PROJECT_DIR/stack.outputs | grep $ARCH-instance-ip | cut -d ' ' -f 2)
+    - MICRO_VM_IP=$(cat $CI_PROJECT_DIR/stack.outputs | grep $ARCH-$TAG | cut -d ' ' -f 2)
+    # ssh into each micro-vm and run initialization script. This script will also run the tests.
+    - ssh -o StrictHostKeyChecking=no -i $AWS_EC2_SSH_KEY_FILE ubuntu@$INSTANCE_IP "ssh -o StrictHostKeyChecking=no -i /home/kernel-version-testing/ddvm_rsa root@${MICRO_VM_IP} 'bash /root/fetch_dependencies.sh && /micro-vm-init.sh ${GOVERSION}'"
```

Comment:
Where is `fetch_dependencies.sh` living?

Reply:
It is added in the micro-vm image:
https://github.com/DataDog/ami-builder/blob/usama.saqib/kernel-version-testing-ami/scripts/kernel-version-testing/make-dev-image.sh#L224
https://github.com/DataDog/ami-builder/blob/usama.saqib/kernel-version-testing-ami/scripts/kernel-version-testing/ubuntu/provision-cloudimg.sh#L184